### ISSUE TITLE
Blocked by Laravel twitter account

### ISSUE THREAD

**Issue by metaunicorn** [laravel/framework#15700]
Hi, first of all I know this is not the proper way to communicate this and issue tracking not for this, but well, I cannot find another communication channel and actually I have an issue: I can't read any tweets from the `@laravelphp` or  @taylorotwell Twitter accounts because they blocked me.

I don't understand the reasons why they made this harsh and out-of-proportions decision. AFAIR I only expressed an opinion the last year about how "hacky" is to achieve code completion with some IDEs and Laravel, because of the design patterns that have been used. Always with respect and in a good way.

People should be able to express their opinions. I like Laravel, but I am not 100% agree with every decision that has been made. That's fair, IMO. You shouldn't block people from access information or prevent them to use your library because they have different opinions in some things...

I don't understand this drama and this "punishment" and I feel very sad that things turned out this way, without even an explanation.

I would appreciate if you (the account managers) could unblock my Twitter account: `@metaunicorn`.

Thanks.

**Comment by metaunicorn** [comment]
@GrahamCampbell are you going to do something about this?

**Comment by GrahamCampbell** [comment]
Nothing to do with me, and this is the wrong place, you are correct. Sorry.

**Comment by metaunicorn** [comment]
ok :(

**Comment by GrahamCampbell** [comment]
I find it a bit immature that you blocked me too. Makes it impossible to review your tweets. On that note, I will not be following up further.

**Comment by metaunicorn** [comment]
@GrahamCampbell I didn't remember that sorry. I blocked back those accounts because I felt a bit upset and I didn't understand the reason so I also blocked other contributors accounts. But I unblocked them some time ago, I might have forgot yours.

**Comment by metaunicorn** [comment]
You are unblocked

**Comment by metaunicorn** [comment]
I gave you the explanation and still got no action from your side, I didn't expect a response but at least I expected a bit of understanding and be unblocked.

Look I am not a spammer, or a guy that makes harsh comments nor one that likes to annoy people. I am a friendly person and all of this is a misunderstanding. I don't deserve this situation.

Anyway it's neither professional nor very mature to block people without a reason, and worse, a Laravel developer using actively the ecosystem, just because he has a different opinion of something.

I used to talk a lot of Laravel and retweet a lot of articles, etc.
Now this is a big turn off.

We both have done mistakes in this situation. Now it's up to you if you want to resolve it or not.

Cheers
Jota
